### PR TITLE
mgr/dashboard: add polish language

### PR DESCRIPTION
**Pull request by dziomdziorae**
Add polish language to the list.

Fixes: https://tracker.ceph.com/issues/38236

- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/test/common/test_blkdev_sys_block/sys/dev/block/8:0 [hunk 1]
```diff
@@ -1 +0,0 @@
-../../block/sda
```
Comment (votdev) [inline src/test/common/test_blkdev_sys_block/sys/dev/block/8:0]: I think this and the other files are added to the PR by accident. Please remove them.

File: src/test/common/test_blkdev_sys_block/sys/dev/block/9:0 [hunk 2]
```diff
@@ -1 +0,0 @@
-../../block/sdb
```
Comment (votdev) [inline src/test/common/test_blkdev_sys_block/sys/dev/block/9:0]: Please remove.

**Comment by LenzGr** [comment]
Thank you for your submission! Can you please amend your git commit message
as follows:
```
mgr/dashboard: Add Polish language

Fixes: https://tracker.ceph.com/issues/38236

Signed-off-by: Elzbieta Dziomdziora <elzbieta.dziomdziora@ts.fujitsu.com>
```
Also, consider using a dedicated branch in your fork instead of the `master` branch - this makes it easier to amend change sets and force-push changes to the remote repository.

File: src/test/common/test_blkdev_sys_block/sys/dev/block/8:0 (hunk 1 above)
Comment (Exotelis) [reply src/test/common/test_blkdev_sys_block/sys/dev/block/8:0]: You might can fix this with a rebase

**Comment by tchaikov** [comment]
@dziomdziorae your commits add up to an empty changeset. could you re-add your change and rebase on to the latest master and repush?

**Comment by votdev** [comment]
Superseded by #27152.

**Pull request closed without merging**